chapters/hpmor-chapter-030.tex

```diff
@@ -543,6 +544,7 @@
 
 And the survivors of the Sunshine Regiment marched off toward the victory field, singing their marching song as they went:
 
+\vskip 0pt plus 4\baselineskip\settowidth{\versewidth}{And send you to a home that’s true,}
```

Comment:
Why is text added here?

Reply:
in all other verses the versewidth was calculated via the longest line of the verse. I applied the same here.